R/meta_regression_plots.R

```diff
@@ -0,0 +1,487 @@
+#' Create a covariate regression plot where multiple comparisons can be plotted, and the contributions from each study are shown as circles.
+#'
+#' @param model GEMTC model result object.
+#' @param treatment_df Reactive containing data frame containing treatment IDs (Number), sanitised names (Label), and original names (RawLabel).
+#' @param reference Name of reference treatment.
```

Comment:
Specify data types for all parameters (and for other functions)

Reply:
I've added a bit more clarification to the roxygen comments. I think that they're sufficiently clear, other than the contribution type, but that is mostly because that functionality hasn't yet been agreed. After this Thursday's meeting, it should be decided.